Functions.Templates/Templates/SqlOutputBinding-JavaScript/index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    var rows = req["body"]?.rows ?? [];
```

Comment:
Given that most of our samples are for GET requests it seems better to also have the templates use that as well (i.e. use the URL parameters instead)

Reply:
can we add multiple templates to the bindings, I thought we can only add one and post is what I used for all the JavaScript output binding examples.

Reply:
I don't believe we can.

Why did you choose POST? Our C# output binding samples all use GET so it'd be better to be consistent with those.

Reply:
It doesn't make sense to use get for post requests (sending data out). It only make sense when we insert one row at a time, when doing multiple rows you have to use post anyway. I wasn't sure why C# samples were created that way but I can create a sample using params as an example and add it to be consistent. But I don't think that's the right way.

Reply:
What do you mean "it doesn't make sense"? You can do both just fine. 

(and you can totally do multiple items with GET, you could just have a data blob that's a serialized array or something. Which at that point I agree using POST is much cleaner but it still *can* be done)

We should be consistent in our examples though - and GET is a lot easier to look at and understandso it makes sense for us to use that for the majority of our samples. But if we wanted to switch I don't really have a huge preference as long as we do the same for both. (and for the Python samples that I've made which mostly use GET)

Reply:
@dzsquared @chlafreniere Any preference here?

Reply:
I agree with @maddydev about using POST on the output binding and apologize for not catching that the C# output binding is GET only.
The default http trigger templates enable both GET and POST, with an example of checking for a request body before opting to a param value. That's a bit too much for our scenario, but it would be nice in the longer term to show output bindings as POST requests.

Reply:
Are you ok with having our samples and templates use the different methods then? Because I am mostly saying here that we should be consistent - not that GET is better than POST in this scenario. Updating the templates/samples to support both seems like a good idea in the medium term though. 

If we're sticking with just POST then you should at least update function.json to indicate it only handles POST requests.